src/views/Editor.vue

```diff
@@ -279,6 +279,7 @@ export default {
 			qvainData: null,
 			reloadDatasetCounter: 0,
 			reloadDatasetTimer: null,
+			otherError: false,
```

Comment:
why we need variable?

Reply:
at the time of redirecting to dataset page from editor, code from method checktab() from mounted and watch lifecycle-hooks is getting called . and this code was causing to show other page (selecting either ida or att project page). so added a variable to skip this flow in case of erroneous condition